### ISSUE TITLE
How to get the pseudo-label confidence?

### ISSUE THREAD

**Issue by Lanezzz** [CVIR/TCL#3]
Thanks for your great work! However,I have met a question.You said in your paper in page5 that " Finally, for finetuning with pseudo-labels, we train our model with both labeled and unlabeled videos having pseudo-label confidence more than 0.8 for 50 epochs" So how to calculate the confidence of  pseudo-label ? And Is 0.8 a empirical parameter?

**Comment by omchakrabarty** [comment]
Yes, we set the `threshold` parameter to `0.8` for considering the predicted labels with confidence higher than `0.8` as the corresponding pseudo-labels.
